code/components/mqtt_ctrl/server_mqtt.cpp

```diff
@@ -878,6 +890,9 @@ static bool publishHADiscoveryTopic(const strHADiscoveryData *_data, const int _
             payload += "\"pl_on\":\"1\",";  // payload "ON"
             payload += "\"pl_off\":\"0\","; // payload "OFF"
         }
+    }else if (_data->topicName == "reboot") {
```

Comment:
Please reuse existing code. It seems identical than cycle_start, therefore add it above with OR. Adapt comment accordingly

I'd would add this button as hidden button per default to avoid pressing it accidentally:

This can be done by adding "enabled_by_default" flag to the payload --> "\"en\": \"false"\,"